src/PKSim.Core/Snapshots/Mappers/QualificationStepMapper.cs

```diff
@@ -74,6 +74,8 @@ private void mapQualificationStepPropertiesToModel(IQualificationStep qualificat
 
                runSimulationQualificationStep.Simulation = simulation;
                break;
+            default:
+               throw new ArgumentException(PKSimConstants.Error.NotMappingDefinedForQualificationStep(qualificationStep.GetType().Name));
```

Comment:
Ok I have updated the switch to add a default case. I cannot test it however because when a type is undefined, the caller exists before already. The only way to get there as mentioned earlier, is to implement something halfway (define a type, and never use it),
That's why a threw an exception as well to get the full stacktrace.